packages/flutter/lib/src/material/text_field.dart

```diff
@@ -278,6 +279,11 @@ class TextField extends StatefulWidget {
   /// [Decoration.enabled] property.
   final bool enabled;
 
+  /// When the keyboard pops up and this [EditableText] is in a scrollable container it will attempt to
+  /// scroll that to make itself visible.
+  /// This value controls how much space should be between this and the keyboard.
+  final double scrollPadding;
```

Comment:
Maybe this should be an EdgeInsets instead of a double to control the padding in each cardinal direction individually (see comment above for context)?